ergo-wallet/src/main/java/org/ergoplatform/wallet/interface4j/SecretString.java

```diff
@@ -0,0 +1,96 @@
+package org.ergoplatform.wallet.interface4j;
```

Comment:
Is this copied from Appkit or it has some difference? Please comment in the code with direct reference if necessary.

Reply:
Noted